internal/build/testdata/TestBuildkitPrinter/add-failure.master.txt

```diff
@@ -1,6 +1,7 @@
-[1/3] FROM docker.io/library/busybox@sha256:5acba83a746c7608ed544dc1533b87c737a0b0fb730301639a0179f9344b1678
+[1/3] FROM docker.io/library/busybox
 [background] read source files
-[background] read source files 362B [done: 35ms]
+[background] read source files 26B
```

Comment:
this line is the important bit - now we print the intermediate state a bit more effectively.